### PR TITLE
Removing booleans.md

### PR DESCRIPTION
**Pull request by sydney-runkle**
## Change Summary

Removing `booleans.md` file and moving content to new `standard_library_types.md` file.

## Related issue number

Contributing to #7322

## Checklist

* [x] The pull request title is a good summary of the changes - it will be used in the changelog
* [ ] Unit tests for the changes exist
* [x] Tests pass on CI
* [x] Documentation reflects the changes where applicable
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Selected Reviewer: @dmontagu

### CODE REVIEW

**Comment by sydney-runkle** [comment]
please review

File: docs/usage/types/standard_types.md
Comment (Kludex) [inline docs/usage/types/standard_types.md:1]: Can we revert the changes in this file? 🤔 

(besides the boolean link change)

**Review by Kludex — CHANGES_REQUESTED** [review]
Just a comment. Everything else looks good. :)

**Comment by Kludex** [comment]
please update

**Review by Kludex — APPROVED** [review]

**Pull request merged**